packages/lsp/test/connection_manager.spec.ts

```diff
@@ -99,5 +100,12 @@ describe('@jupyterlab/lsp', () => {
         expect(manager.disconnectDocumentSignals).toHaveBeenCalled();
       });
     });
+    describe('#dispose()', () => {
+      it('should dispose of an uninitialized connection without errors', () => {
+        const connection = new LSPConnection();
```

Comment:
The test is failing because `LSPConnection` constructor requires `options` argument.

Reply:
Fixed with the latest commits.